src/components/form/adapter/base-adapter.ts

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
```

Comment:
Apart from some stuff like the added rule for prettier (odd, I thought we already had a rule for trailing commas 🤔), this seems to be one new feature, so I don't think there should be 17 separate commits. Any fixes of bugs in the new features should be squashed into the feature commit for example. Since the feature was never released with the bug, there's no reason for a separate changelog entry for the bug fix, nor is there any need for a separate commit for that fix.

I haven't looked at this in detail yet, I'm really just trying to catch up on a bunch of emails from the last few days, and saw this. I'll put it in review on our board, and someone else might review it, but I wanted to add this comment 🙂

Reply:
@adrianschmidt You can take this off the board for now i've moved it back to a draft commit I have to update this PR based on changes and feedback in https://github.com/Lundalogik/lime-elements/pull/781